### DISCUSSION TITLE
Rectangle shaped keyrunes

### DISCUSSION THREAD

**Discussion by yandere-sliver** [andrewgioia/keyrune#255] [General]
So I looked through a bunch of option and came to the conclusion that it is probably an issue with the glyphs themselves and not the css settings.

It seems that keyrunes assumes that a set symbol is roughly the size of a square. 
You can see that here: https://mtg.gamepedia.com/Set

While there are in general many square shaped set symbols the dimensions of a set symbol are more akin to a rectangle with double the width of the height. You can see that quite well on the promotional star: https://scryfall.com/card/j14/8/%7Cducgnorn-gsedzaonobt 

I wonder if there is some possibility to get rid of the whitespace on top and bottom on the more rectangle shaped keyrunes: ARN, ATQ, FEM, ALL, WTH, EXO, UGL, EXO, USG, ATH, ULG, BRB, NEM, BTD, LGN, MRD, EVG, SHM, M10, M11, M12, PC2, M13, V12, GTC, DGM, M14, M15, DDN, C14, FRF, TPR, V15, C15, W16, EMN, PCA, W17, UST, M19, MED, GRN, UMA, RNA, MH1, M20, UND, M21, 2XM, CC1, PMEI, PGRU, and I probably missed a few.

Or to put it differently I would like to have an option to set the height of the symbol and then have the symbol display at its actual height and not half its height. 

I am not sure if this is possible with the current setup.

**Comment by andrewgioia** [comment]
@yandere-sliver these are excellent points and unfortunately arise from limitations with displaying the symbols as fonts and in an original attempt to normalize them height-wise. 

On the actual paper cards, set symbols can be any dimension and some are often much wider than they are tall. To display every symbol properly as similarly-sized font glyphs I had originally fixed the height of every glyph (512px). The narrow symbols can be thinner than 512 but I had also originally set a max-width of that for them, creating some very small-looking icons for symbols that are short and wide.

I think in retrospect this was incorrect and the proper fix is to redo them all (😬) with 2 key changes: widths can be variable and go beyond a square maximum, and each symbol should be sized by eye so that they all appear to be similarly sized visually. This is is my intent for a v4 that would include all new SVGs and am very much open to suggestions or recommendations from anyone using this or who has experience with other similar packages.

**Comment by andrewgioia** [comment]
Moving this (and all older v4 issues) to discussions!
